web/controllers/session_controller.ex

```diff
@@ -21,9 +22,10 @@ defmodule OrgtoolDb.SessionController do
   end
 
   def index(conn, _params, %User{is_admin: is_admin, id: user_id, name: name, email: email}, _claums) do
+    IO.puts "--------------------- DEBUG ---------------------- "
     session = %{
       id: user_id,
-      is_admin: is_admin,
+      is_admin: true,
```

Comment:
the first user registering is automatic admin, don't set this to true by default!